### DISCUSSION TITLE
Ender 3 V2 heating problems

### DISCUSSION THREAD

**Discussion by Basmes23** [mriscoc/Ender3V2S1#1327] [Q&A]
Hello, I did the 300C update for my printer. I then entered the command C104 U1 T315 and C500 via pronterface. Now I can also reach temperatures of 300C, but after 300C the temperature continues to rise until 312C and then an error message appears. If I follow the temperature in Pronterface, the error appears in the display as soon as the value has risen above 300C. I dont know how i can fix this problem. Should i have to do a pid tunig?
![Uploading 20240711_160735.jpg…]()

**Comment by mriscoc** [comment]
Yes, you always must follow a calibration routine: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
Check if your firmware has MPC enabled, in that case, you need to perform a MPC tune.
